src/main/kotlin/com/devfloor/untitled/favorite/domain/FavoriteType.kt

```diff
@@ -0,0 +1,7 @@
+package com.devfloor.untitled.favorite.domain
+
+enum class FavoriteType(val type: String) {
+    FAVORITE("좋아요"),
+    WONDER("나도 궁금해요"),
+    CLIP("스크랩"),
+}
```

Comment:
```suggestion
enum class FavoriteType(val text: String) {
    FAVORITE("좋아요"),
    WONDER("나도 궁금해요"),
    CLIP("스크랩"),
}
```

이거 이렇게 고치면 어떨까? 내꺼는 이렇게 바꿔놨는데 enum 에서 한글 설명? 한글 이름? 이라고 보이는 프로퍼티들을 공통적으로 text로 가져가면 좋을 것 같아. 이렇게 갑자기 생각이 든 게 나중에 favorite에서 따라서 조회해보면 기존에는 favorite.type.type 인데 이렇게 바꾸면 favorite.type.text여서 조금 더 가독성이 좋을 것 같다는 생각이 들었어!! ㅋㅋㅋ

Reply:
> favorite.type.type 인데 이렇게 바꾸면 favorite.type.text여서 조금 더 가독성이 좋을 것 같다는 생각이 들었어!! ㅋㅋㅋ

오키 option부분도 classNumber -> text 로 해서 아예 enum에서는 text로 일관하는거로!

> 아 그리고 테스트 코드가 하나도 없는데 이번주 남은 시간동안 테스트 코드도 추가해주면 좋을 것 같아!!

테스트.. 가능할까 ㅋㅋㅋ 일단 구현부분이 전부 완성되면 테스트를 짜야할거가튼디

Reply:
굿굿!! 테스트는 우리 다 짜는걸로 하자! 이거 기능 개발도 중요하지만 테스트 코드 빨리 짜는 연습도 한다고 생각하고 ㅋㅋ